### PR TITLE
PR: Fixed E2E test, template compiles and index file

### PR DESCRIPTION
**Pull request by chrislaughlin**
I have added actual E2E tests for out application all working. 
Fixed the template compile now compiling our templates. 
Fixed karma config to allow pass this will need to be fixed up for our unit tests.

### CODE REVIEW

**Comment by chrislaughlin** [comment]
The last pull request got messed up as I forgot to pull before pushing

**Comment by martinmicunda** [comment]
yeah i can see that :)

**Comment by chrislaughlin** [comment]
Need to update the build as the jshint file was moved

File: gulpfile.js [hunk 1]
```diff
@@ -231,7 +232,7 @@ gulp.task('develop', function () {
     };
 
     nodemon(options)
-        .on('change', ['jshint:server'])
+        //        .on('change', ['lint'])
```
Comment (martinmicunda) [inline gulpfile.js]: the all file seems to be bad merged...

Comment (chrislaughlin) [inline gulpfile.js]: What should that line look like

Comment (martinmicunda) [inline gulpfile.js]: not just line all file.. U r overwriting previous commit...

Comment (martinmicunda) [inline gulpfile.js]: all lines there are removed by your commit should be there and lines that u r adding should not be there

Comment (chrislaughlin) [inline gulpfile.js]: I think web storm auto mergered the file

Comment (chrislaughlin) [inline gulpfile.js]: ill fix it up now

**Comment by chrislaughlin** [comment]
Passed the build so just needs someone to review and merge.

**Comment by chrislaughlin** [comment]
Thanks I'll make sure to merge correctly next time

On Sunday, June 1, 2014, Martin Micunda notifications@github.com wrote:

> Merged #21 https://github.com/cam-technologies/time-booker/pull/21.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cam-technologies/time-booker/pull/21#event-126829014.

## 

Regards

Chris Laughlin

**Comment by chrislaughlin** [comment]
Wasn't expecting any changes as there was no recent pull requests

On Sunday, June 1, 2014, Christopher Laughlin bluntm@gmail.com wrote:

> Thanks I'll make sure to merge correctly next time
> 
> On Sunday, June 1, 2014, Martin Micunda <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
> 
> > Merged #21 https://github.com/cam-technologies/time-booker/pull/21.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/cam-technologies/time-booker/pull/21#event-126829014
> > .
> 
> ## 
> 
> Regards
> 
> Chris Laughlin

## 

Regards

Chris Laughlin

**Comment by martinmicunda** [comment]
np ;)

**Pull request merged**